src/frontends/tensorflow_common/src/op/identity.cpp

```diff
@@ -14,7 +14,13 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_identity_op(const NodeContext& node) {
-    vector<string> supported_ops = {"Identity", "PreventGradient", "Snapshot", "StopGradient"};
+    vector<string> supported_ops = {"Identity",
+                                    "PreventGradient",
+                                    "Snapshot",
+                                    "StopGradient",
+                                    "ReadVariableOp",
+                                    "ShardedFilename",
```

Comment:
I think for `ShardedFilename` we need to implement separate translator. We expect `StringConstant` input there and this translator will replace `ShardedFilename` with the new `StringConstant` that is computed as `%s-%05d-of-%05d, basename, shard, num_shards` according to its specification. Let make it in the straightforward way.

Reply:
Right now it is unused. As I understand - leave it as is right now.